From 6ff81e72ae8319b37f1c331f2543fe288b0fa471 Mon Sep 17 00:00:00 2001
Message-ID: <6ff81e72ae8319b37f1c331f2543fe288b0fa471.1707833700.git.sharlatanus@gmail.com>
In-Reply-To: <9c3d89f5eb0359dc6493dfc0d24667d2a5c6b484.1707833700.git.sharlatanus@gmail.com>
References: <9c3d89f5eb0359dc6493dfc0d24667d2a5c6b484.1707833700.git.sharlatanus@gmail.com>
From: Felix Gruber <felgru@posteo.net>
Date: Mon, 12 Feb 2024 18:04:09 +0000
Subject: [PATCH v4 2/8] gnu: python-mdit-py-plugins: Update to 0.4.0.

* gnu/packages/python-xyz.scm (python-mdit-py-plugins): Update to 0.4.0.

Change-Id: I18ca46abebe125d1a6729c10a3f4fa4a62a145ba
Signed-off-by: Sharlatan Hellseher <sharlatanus@gmail.com>
---
 gnu/packages/python-xyz.scm | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnu/packages/python-xyz.scm b/gnu/packages/python-xyz.scm
index 6a28836acf..30f6f1fb3e 100644
--- a/gnu/packages/python-xyz.scm
+++ b/gnu/packages/python-xyz.scm
@@ -1124,7 +1124,7 @@ (define-public python-lunr
 (define-public python-mdit-py-plugins
   (package
     (name "python-mdit-py-plugins")
-    (version "0.3.1")
+    (version "0.4.0")
     (source (origin
               (method git-fetch)        ;for tests
               (uri (git-reference
@@ -1133,7 +1133,7 @@ (define-public python-mdit-py-plugins
               (file-name (git-file-name name version))
               (sha256
                (base32
-                "00ybbfb6dzn9q5qnvnm8bmhd84s215ik7pcdb4r35zhwiv2ikizz"))))
+                "1790fcf3nd9c5j37fx87x3ivfyihk6l74jxj1qxvw3qfyb96w4k0"))))
     (build-system pyproject-build-system)
     (propagated-inputs
      (list python-markdown-it-py
-- 
2.41.0

